src/HttpClient/Guzzle6HttpClient.php

```diff
@@ -4,9 +4,9 @@
 
 use Crell\ApiProblem\ApiProblem;
 use eLife\ApiSdk\Exception\ApiException;
-use eLife\ApiSdk\Exception\ApiProblemException;
-use eLife\ApiSdk\Exception\NetworkException;
-use eLife\ApiSdk\Exception\ResponseException;
+use eLife\ApiSdk\Exception\ApiProblemResponse;
+use eLife\ApiSdk\Exception\NetworkProblem;
+use eLife\ApiSdk\Exception\BadResponse;
```

Comment:
Ah, we should enable that everywhere.

Shame about the `use function` though, wonder if that's an upstream bug or intentional.